fr.json

```diff
@@ -1030,12 +1030,12 @@
   "deleteMyAccount": "Supprimer mon compte",
   "emailUs": "Envoyez-nous un email",
   "languageAndUnits": "Langue et unités",
-  "addEngineRun": "Ajouter un moteur en marche",
-  "engineRun": "Moteur en marche",
+  "addEngineRun": "Ajouter un temps moteur",
+  "engineRun": "Temps moteur",
   "reset": "Réinitialiser",
-  "selfLaunchEnd": "Fin du lancement automatique",
-  "towRelease": "Libération de l'avion",
-  "winchRelease": "Libération de l'avion",
+  "selfLaunchEnd": "Fin du lancement autonome",
+  "towRelease": "Largage remorqueur",
+  "winchRelease": "Largage treuil"
```

Comment:
Thanks for the updated translations! Here is a "," missing at the end of the line according to the (newly added) syntax checker.